tests/integration/security/authz_test.go

```diff
@@ -954,6 +954,78 @@ func TestAuthz_PathPrecedence(t *testing.T) {
 		})
 }
 
+func TestAuthz_PathTemplating(t *testing.T) {
```

Comment:
Currently failing for all non a to b tests (fails for tproxy, statefulset, headless, etc.)

Reply:
The pattern `/allow/admin/{**}` will be translated by istio to  `/allow/admin/**`. The path `/allow/admin` will not match on the pattern because envoy is expecting a trailing slash, `/allow/admin/`. I believe this is an Envoy bug and I am working on the fix.

Reply:
https://github.com/envoyproxy/envoy/issues/33571

Reply:
I think this is actually the expected behavior. Resolving for now.